config/config.json

```diff
@@ -1,8 +1,8 @@
 {
   "development": {
-    "username": "ela",
+    "username": "marcocampos",
```

Comment:
get this from process.ENV.DB_USERNAME because william has a different username in his own local computer: https://github.com/izelnakri/nycda-blog-application/commit/e929a158f3f3f3d20f63b853ad0af91986f9772f

you need to add DB_USERNAME environment variable to your ~/.bash_profile